_includes/app/BasicSample.java

```diff
@@ -0,0 +1,53 @@
+import java.sql.*;
+
+/*
+You can compile and run this example with a command like:
```

Comment:
That format is specific to documentation comments (i.e. javadoc), and I don't think I'd call this a javadoc comment.